zeppelin-interpreter-integration/src/test/java/org/apache/zeppelin/integration/ZeppelinSparkClusterTest21.java

```diff
@@ -34,7 +34,7 @@ public ZeppelinSparkClusterTest21(String sparkVersion) throws Exception {
   @Parameterized.Parameters
   public static List<Object[]> data() {
     return Arrays.asList(new Object[][]{
-            {"2.1.2"}
+            {"2.1.3"}
```

Comment:
can this be passed in thru travis? 

and should these be updated https://github.com/apache/zeppelin/blob/master/.travis.yml#L136 "2.1.0"

https://github.com/apache/zeppelin/blob/master/.travis.yml#L142 "2.2.0"

Reply:
These 2 are livy interpreter, not related here

Reply:
ok for livy is there new version support https://github.com/apache/zeppelin/blob/master/.travis.yml#L142?

Reply:
0.5.0 is the latest livy release, livy community is discussing release of 0.6.0